AnimationCodelab/finished/build.gradle

```diff
@@ -44,7 +44,7 @@ android {
 }
 
 dependencies {
-    implementation 'androidx.core:core-ktx:1.6.0-alpha01'
+    implementation 'androidx.core:core-ktx:1.6.0-beta02'
```

Comment:
Update the `activity-compose` dependency below to `beta01`. And also in the start module. Thanks!